### PR TITLE
refactor(handle): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1664 #1500

### CODE REVIEW

File: src/components/calcite-handle/calcite-handle.scss [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-  border: none;
-  color: var(--calcite-ui-border-3);
+  @apply flex 
+    items-center 
+    self-stretch 
+    content-center 
+    bg-transparent 
+    text-color-2 
+    bg-foreground-3 
+    bg-transparent
```
[the remark this answers is not in the document]
Comment (caripizza) [reply src/components/calcite-handle/calcite-handle.scss:13]: wait, which background color should it have, `bg-foreground-3` or `bg-transparent`?

File: src/components/calcite-handle/calcite-handle.scss [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+    self-stretch 
+    content-center 
+    bg-transparent 
+    text-color-2 
+    bg-foreground-3 
+    bg-transparent 
+    border-none 
+    cursor-move;
+  padding: theme("spacing.3") theme("spacing.1");
+  @apply cursor-move;
```
Comment (caripizza) [inline src/components/calcite-handle/calcite-handle.scss:17]: remove duplicate `cursor-move`

File: src/components/calcite-handle/calcite-handle.scss [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
   line-height: 0;
-  cursor: move;
   &:hover {
-    background-color: var(--calcite-ui-foreground-2);
-    color: var(--calcite-app-foreground);
+    @apply bg-foreground-2 text-color-2;
   }
-  @include focus-style-base();
+  @apply focus-base;
```
Comment (caripizza) [inline src/components/calcite-handle/calcite-handle.scss:22]: move `focus-base` into the existing @apply rule above

**Review by caripizza — CHANGES_REQUESTED** [review]

File: src/components/calcite-handle/calcite-handle.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (julio8a) [reply src/components/calcite-handle/calcite-handle.scss:13]: I would think transparent since that's the last one.

**Comment by paulcpederson** [comment]
I, for one, agree with everything @caripizza said and that is my review.

File: src/components/calcite-handle/calcite-handle.scss [hunk 4]
```diff
@@ -1,30 +1,29 @@
...
-  justify-content: center;
-  padding: var(--calcite-spacing-three-quarters) var(--calcite-spacing-quarter);
-  background-color: var(--calcite-ui-foreground-3);
-  background-color: transparent;
-  border: none;
-  color: var(--calcite-ui-border-3);
+  @apply flex 
+    items-center 
+    self-stretch 
+    content-center
```
[the remark this answers is not in the document]
Comment (julio8a) [reply src/components/calcite-handle/calcite-handle.scss]: done

File: src/components/calcite-handle/calcite-handle.scss (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-handle/calcite-handle.scss]: done

File: src/components/calcite-handle/calcite-handle.scss (hunk 3 above)
Comment (julio8a) [reply src/components/calcite-handle/calcite-handle.scss]: good catch, done

**Review by paulcpederson — APPROVED** [review]

**Review by caripizza — APPROVED** [review]

**Comment by julio8a** [comment]
Yes! I've submitted 10 PR's. It's going to feel good to hit the merge button!

**Pull request merged**